### PR TITLE
IMP-28 Importer Project - API

### PR DESCRIPTION
**Pull request by sam-ludlow**
https://io1dev.atlassian.net/browse/IMP-28

replaced by https://github.com/invotra/api/pull/333

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -3386,6 +3432,18 @@ paths:
...
             type: string
             example: '2012-12-25'
+        - description: 'Filter by external ID.'
+          in: query
+          name: external_id
+          schema:
+            type: string
+            example: 'ABC123'
+        - name: state
+          description: 'Filter the list of lists on a comma-separated list of states. Valid statuses are "draft", "needs_review", "reviewed", "published", and "unpublished". You may include multiple. The default when not supplied is "published" only.'
```
Comment (ParryLuke) [inline api.swagger.yaml:3442]: Not sure this should be in this PR? If it should then 'lists' should be 'ideas'

File: api.swagger.yaml [hunk 2]
```diff
@@ -5018,6 +5118,18 @@ paths:
...
             type: string
             example: '2012-12-25'
+        - description: 'Filter by external ID.'
+          in: query
+          name: external_id
+          schema:
+            type: string
+            example: 'ABC123'
+        - name: state
+          description: 'Filter the list of webforms on a comma-separated list of states. Valid statuses are "draft", "needs_review", "reviewed", "published", and "unpublished". You may include multiple. The default when not supplied is "published" only.'
```
Comment (ParryLuke) [inline api.swagger.yaml:5128]: Same as above - should it be here / should it say support tickets?

**Review by ParryLuke — CHANGES_REQUESTED** [review]

File: api.swagger.yaml (hunk 1 above)
Comment (sam-ludlow) [reply api.swagger.yaml]: chopped out

File: api.swagger.yaml (hunk 2 above)
Comment (sam-ludlow) [reply api.swagger.yaml]: chopped out

**Review by ParryLuke — APPROVED** [review]

**Comment by sam-ludlow** [comment]
replaced by https://github.com/invotra/api/pull/333

**Pull request closed without merging**